k8s/bases/cse-export-d2cc/cse-export-d2cc-job-launcher-configmap.yaml

```diff
@@ -51,3 +51,4 @@ data:
       url:
         task-manager-timestamp-url: http://cse-export-d2cc-task-manager/tasks/
         task-manager-business-date-url: http://cse-export-d2cc-task-manager/tasks/businessdate/
+        interrupt-run-url: http://interruption-server/interrupt/
```

Comment:
Is there nothing to change in the Ingress configurations to make the interruption server reachable from other containers?

Reply:
not that I noticed during azure testing